src/app/data-model/Nullable.h

```diff
@@ -80,6 +80,16 @@ struct Nullable : protected Optional<T>
         return true;
     }
 
+    // Set the nullable to the `other` nullable, returning true if something actually changed.
+    // This can be used to determine if changes occurred on assignment, so that reporting can be triggered
+    // only on actual changes.
+    constexpr bool SetToMatch(const Nullable<T> & other)
```

Comment:
This name feels odd ... are there libraries that use this name for reference? I would have expected `Assign` or `Update` or even just `Set`.

Reply:
Renaming to Update